packages/react/src/internal/selectedItems/SelectedItems.module.scss

```diff
@@ -32,6 +32,19 @@
     }
   }
 
+  &.truncateFirstChild {
+    flex-wrap: nowrap;
+    & > :first-child {
+      justify-content: flex-end;
+      overflow: hidden;
+      > span {
```

Comment:
It's a shame that the span inside Tag component doesn't have a class and styles because browsers parse the selectors bottom-up, meaning the selectors are read from right to left. So first the browser finds all the span elements in the DOM and then it continues with the selectors. Should we refactor the Tag component and give a prop to Tag's span element?

Reply:
Could we use the Tag's className for this?

Reply:
With hds-prefix, `.hds-dropdown-tag` etc?

Reply:
I actually tried to pass className to <Tag> via "labelProps", but that overrides all tag's classNames. So I rejected that approach, because tag loses all base styles. 

How can I pass "hds-dropdown-tag" to the Tag-component?

Reply:
I guess a new props like `labelClassName` would need to made to Tag component. I'm not sure if the styling needs to be at the innermost span or the "label" span that wraps it? If the wrapper is enough => prop `labelClassName` and merge that with the existing className. If the innermost, then the naming gets a bit confusing imo. Maybe `textClassName` ?

Reply:
This is to get rid of `> span` and replace it with `> .hds-dropdown-tag`. For performance gain?

Reply:
And also for clarity and for better selector 👍 
The class tells more than span that we are targeting the "tag" 😁

Reply:
Done.